### PR TITLE
Pre cloud

### PR DESCRIPTION
**Pull request by teska77**
Caching is the devil

### CODE REVIEW

**Review by kchokanov — DISMISSED** [review]
No New Line at the end of a file

File: Application/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DB_USER=safeuser
+DB_PASS=password
+DB_NAME=api_db
+DB_HOST=db
```
Comment (Dmium) [inline Application/.env]: NEWLINE

Comment (Dmium) [reply Application/.env]: PLS

**Review by Dmium — DISMISSED** [review]

**Review by Dmium — APPROVED** [review]

**Review by kchokanov — COMMENTED** [review]
NEw LIne at the end of a file :)

**Pull request merged**